docs/sources/articles/configuring.md

```diff
@@ -96,3 +118,84 @@ These options :
 7. Verify that the `docker` daemon is running as specified wit the `ps` command.
 
 		$ ps aux | grep docker | grep -v grep
+
+### Logs
+
+By default logs for `Upstart` jobs are located in `/var/log/upstart` and the logs for `docker` daemon
+can be located at `/var/log/upstart/docker.log`
+
+    $ tail -f /var/log/upstart/docker.log
+    INFO[0000] Loading containers: done.
+    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
+    INFO[0000] +job acceptconnections()
+    INFO[0000] -job acceptconnections() = OK (0)
+    INFO[0000] Daemon has completed initialization
+
+
+## CentOS / Red Hat Enterprise Linux / Fedora
+
+As of `7.x`, CentOS and RHEL  use `systemd` as the process manager. Also as of `21`, Fedora uses
+`systemd` as it's process manager.
+
+After successfully installing Docker for [CentOS](/installation/centos/)/[Red Hat Enterprise Linux]
+(/installation/rhel/)/[Fedora](/installation/fedora), you can check the running status in this way:
+
+    $ sudo systemctl status docker
+
+### Running Docker
+
+You can start/stop/restart `docker` using
+
+    $ sudo systemctl start docker
+
+    $ sudo systemctl stop docker
+
+    $ sudo systemctl restart docker
+
+If you want Docker to start at boot, you should also:
+
+    $ sudo systemctl enable docker
+
+
+### Configuring Docker
+
+You configure the `docker` daemon in the `/etc/sysconfig/docker` file on your
+system. You do this by specifying values in a variable. For CentOS 7.x and RHEL 7.x, the name
+of the variable is `OPTIONS` and for CentOS 6.x and RHEL 6.x, the name of the variable is
+`other_args`. For this section, we will use CentOS 7.x as an example to configure `docker`.
+
+To configure Docker options:
+
+1. Log into your system as a user with `sudo` or `root` privileges.
+
+2. If you don't have one, create the `/etc/sysconfig/docker` file in your system. 
+
+	Depending on how you installed Docker, you may already have this file.
+
+3. Open the file with your favorite editor.
+
+		$ sudo vi /etc/sysconfig/docker
+		
+4. Add a `OPTIONS` variable with the following options. These options are appended to the
+`docker` daemon's run command.
+
+	``` 
+	 OPTIONS=" --dns 8.8.8.8 --dns 8.8.4.4 -D --tls=false -H tcp://0.0.0.0:2375 "
+	```
```

Comment:
Also, these settings should really be discouraged! This will make the docker daemon listen on *all* interfaces, without TLS enabled.

I don't think we should "promote" this by using it as an example in the documentation. Can we come up with something else to illustrate what's possible?